django_rest/tests/test_employees.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+from django.urls import reverse
+from django_rest.models import Employee
+from django.contrib.auth.models import User
+from rest_framework.test import APIClient
+
+
+@pytest.fixture
+def admin_client(api_client=APIClient()):
+    user = User.objects.create_user(username='vololo122', password='gavanava', is_staff=True, is_active=True, is_superuser=True)
+    Employee.objects.create(user=user, birthdate='1988-12-12')
+    resp = api_client.post(path=reverse('token_obtain_pair'), data={'username': 'vololo122', 'password': 'gavanava'}, format='json')
+    token = f"Bearer {resp.data['access']}"
+    api_client.credentials(HTTP_AUTHORIZATION=token)
+    return api_client
+
+
+@pytest.mark.django_db
+def test_get_employee(admin_client):
+    url = reverse('employee_get', args='1')
+    resp = admin_client.get(path=url, data={'pk': '1'})
+    assert resp.status_code == 200
```

Comment:
You should check the response body also, for example, check if `username` is correct.

Reply:
Also it's not the scope of current story.